cpp/include/cudf/utilities/prefetch.hpp

```diff
@@ -31,17 +31,17 @@ namespace detail {
 /**
  * @brief A singleton class that manages the prefetching configuration.
  */
-class PrefetchConfig {
+class prefetch_config {
  public:
-  PrefetchConfig& operator=(const PrefetchConfig&) = delete;
-  PrefetchConfig(const PrefetchConfig&)            = delete;
+  prefetch_config& operator=(const prefetch_config&) = delete;
+  prefetch_config(const prefetch_config&)            = delete;
 
   /**
    * @brief Get the singleton instance of the prefetching configuration.
    *
    * @return The singleton instance of the prefetching configuration.
    */
-  static PrefetchConfig& instance();
+  static prefetch_config& instance();
```

Comment:
`instance` sounds like "creating a new instance" instead of returning a singleton. `get()` is a better name for this usage.

Reply:
But I see that we have use case like `.instance().get(key)` thus calling this `get()` looks too awkward :smile: